lib/pagination.js

```diff
@@ -0,0 +1,30 @@
+const _ = require('lodash')
+
+async function paginate(queryFn, query, variables, dataPath) {
```

Comment:
I will add some documentation to this function if we ever want to reuse it somewhere else :)